<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/settings_FILL0_wght400_GRAD0_opsz24.png</normaloff>img/settings_FILL0_wght400_GRAD0_opsz24.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Обновление (загрузка) базы</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="11" column="0" colspan="2">
        <widget class="QCheckBox" name="update">
         <property name="text">
          <string>Обновлять свежими данными существующие объекты (замедляет при полном сканировании и большой базе)</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QCheckBox" name="full_scan">
         <property name="text">
          <string>Полностью просканировать сайт (иначе остановиться, как только следующая книга найдется в базе)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Строка подключения:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="connection_string"/>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Диапазон сканирования страниц:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="page_start">
           <property name="placeholderText">
            <string extracomment="Первая"/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="page_stop">
           <property name="placeholderText">
            <string extracomment="Последняя"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="12" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Загрузка книг</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Браузер</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Имена каталогов:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="books_dir"/>
       </item>
       <item row="3" column="4">
        <widget class="QComboBox" name="books_browser"/>
       </item>
       <item row="2" column="4">
        <widget class="QComboBox" name="books_naming_method"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>Каталог загрузки:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="books_download_method"/>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="books_dir_select">
         <property name="text">
          <string>Выбрать</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>img/folder_open_FILL0_wght400_GRAD0_opsz24.png</normaloff>img/folder_open_FILL0_wght400_GRAD0_opsz24.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Способ загрузки:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>page_start</tabstop>
  <tabstop>page_stop</tabstop>
  <tabstop>update</tabstop>
  <tabstop>books_dir</tabstop>
  <tabstop>books_dir_select</tabstop>
  <tabstop>books_download_method</tabstop>
  <tabstop>books_naming_method</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>books_dir_select</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>on_books_dir_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>722</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>write_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_books_dir_select()</slot>
  <slot>write_settings()</slot>
 </slots>
</ui>
